hat/assets/js/apps/Iaso/domains/projects/config.tsx

```diff
@@ -116,9 +119,26 @@ export const useFeatureFlagColumns = (
                                 placement="left-start"
                                 arrow
                             >
-                                <HelpOutlineIcon color="primary" />
+                                {settings.row.original.is_dangerous ? (
+                                    <WarningAmberIcon color="warning" />
+                                ) : (
+                                    <HelpOutlineIcon color="primary" />
+                                )}
                             </Tooltip>
                         </Box>
+                    ) : (
+                        <Button
+                            variant="text"
+                            onClick={() =>
+                                toggleFeatureGroup(settings.row.original.code)
+                            }
+                        >
+                            {settings.row.original.collapsed ? (
+                                <ExpandMore />
+                            ) : (
+                                <ExpandLess />
+                            )}
+                        </Button>
```

Comment:
That's a lot of ternaries. I would move that to a separate component and make the code easier to read, even if it gets more verbose